src/App.tsx

```diff
@@ -15,47 +16,111 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SORT_BY_ALPHABET = 'aplhabet';
+const SORT_BY_LENGTH = 'length';
+
+function getPreparedGoods(
+  goods: string[],
+  sortField: string,
+  reverse: boolean,
+):string[] {
+  const copy = [...goods];
+
+  if (sortField) {
+    copy.sort((curGood, nextGood) => {
+      switch (sortField) {
+        case SORT_BY_ALPHABET:
+          return curGood.localeCompare(nextGood);
+        case SORT_BY_LENGTH:
+          return curGood.length - nextGood.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (reverse) {
+    copy.reverse();
+  }
+
+  return copy;
+}
+
 export const App: React.FC = () => {
+  const [sortType, setSortType] = useState('');
```

Comment:
It is suitable to use default value from the enum:
```ts
const [sortType, setSortType] = useState(SortField.Default);

Reply:
And later in your code, when you pass the parameters to setSortType, etc. (lines 57, 65, 70, 78, 83), values from the enum are super handy)

Reply:
agree with [comments](https://github.com/mate-academy/react_list-of-goods-hooks/pull/1107/files#r1258518899)